### PR TITLE
Detach

### PR DESCRIPTION
**Pull request by Tim-Jordison**

### CODE REVIEW

**Comment by tjukes** [comment]
I think it would be nice to see an error response as well. Otherwise once the apiary file has been recompiled this looks fine to me.

File: apiary-sources/endpoints/contacts-values-detach.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+### Detach a Value from a Contact [POST /contacts/{id}/values/detach]
```
Comment (pushplay) [inline apiary-sources/endpoints/contacts-values-detach.md]: This needs to be referenced in api-sections/contacts.md.

**Comment by dano-giftbit** [comment]
Does detach support either a `code` or `valueId` attribute like attach does?

**Comment by pushplay** [comment]
No just valueId.  This was discussed and there was a technical reason we deferred supporting code but I don't recall the reason right now.

**Pull request merged**